examples/dt/neutron/CRD_files/neutron_ovn.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+    spec:
+      neutron:
+        template:
+        customServiceConfig: |
+        [DEFAULT]
+        dns_domain = openstackgate.local
+        vlan_transparent = true
+        service_plugins = qos,ovn-router-flavors-ha,trunk,segments,port_forwarding,log
```

Comment:
IMO these service_plugins settings should be in ipv6 job, i.e the one with service_provider=L3_ROUTER_NAT:user-defined:neutron.services.ovn_l3.service_providers.user_defined.UserDefined
e.g.  ipv6 job http://rhos-ci-logs.lab.eng.tlv2.redhat.com/logs/rcj/DFG-network-neutron-17.1_director-rhel-virthost-3cont_2comp-ipv6-all-geneve-dvr-stf-rally/57/controller-0/var/lib/config-data/puppet-generated/neutron/etc/neutron/neutron.conf.gz

vlan-tenant job does not have these , e.g.

http://rhos-ci-logs.lab.eng.tlv2.redhat.com/logs/rcj/DFG-network-neutron-17.1_director-rhel-virthost-3cont_2comp_1ipa-ipv4-vlan-tls-dvr-ovn/177/controller-0/var/lib/config-data/puppet-generated/neutron/etc/neutron/neutron.conf.gz

Reply:
Fixed this config